Ruth/Chapter_1/exercise_1.12.6

```diff
@@ -0,0 +1,11 @@
+Python 2.7.12 (default, Dec  4 2017, 14:50:18) 
+[GCC 5.4.0 20160609] on linux2
+Type "copyright", "credits" or "license()" for more information.
+>>> 6 + 4 * 9
+42
+>>> 
+==================== RESTART: /home/riri/exercise_1.12.6 ====================
```

Comment:
Write a script to demonstrate this as stated, I know you had to change content to show both but I need to see the script. The name of the script should be `display_eval.py`. :)